das/das.go

```diff
@@ -29,6 +31,8 @@ type DataAvailabilityService interface {
 type DataAvailabilityConfig struct {
 	Enable bool `koanf:"enable"`
 
+	RequestTimeout time.Duration `koanf:"request-timeout"`
```

Comment:
It seems safer to have this be a number of seconds, and put "Seconds" in its name. (time.Duration is a count of nanoseconds).  It can be converted to a `time.Duration` when needed, by multiplying by `time.Second`.

Reply:
Done.

Reply:
I disagree with this since we're using `time.Duration` for a lot of other config options and IIRC the config option parser handles strings with prefixes like '5m', '10s', etc